Update asset.ts - fix file path translation from Windows > Linux

Donovoi opened the pull request:
Some of my folders in Windows had special characters like "@" which were not being handled correctly and stopped everything else from being processed. This uses the library "upath" to fix the issue. I'm not a typescript developer so please let me know if I'm doing something incorrect. 

Thank you

Donovoi committed:
Update asset.ts - fix file path translation from Windows > Linux

JW-CH commented:
Can you update your PR? The build fails and you have some strange changes in the imports etc (changed " to ') etc. Please fix the build errors :)

Donovoi committed:
Merge branch 'immich-app:main' into main

Commit:
Fix issues with reassigning faces and rename in server and web (#9265)

This commit fixes the issues related to reassigning faces in the server and web components. It includes a fix for reassiging faces and a rename operation.

Donovoi commented:
HI I believe I sorted the issues, but how do I test if the build is ok? I've tried to follow 
https://immich.app/docs/developer/pr-checklist
I also ran make e2e and it seems to run

Donovoi commented:
Sweet thank you! Let me know when you do and I'll fix up any issues 😁

zackpollard commented:
You seem to have an extra dependency "install" that is not required, also the builds and tests are failing. Can you fix these two problems please?

Donovoi committed:
Merge branch 'immich-app:main' into main

Commit:
chore: Update npm dependencies

Update npm dependencies to the latest stable versions, including "install" and "upath".

Co-authored-by: Tushar Harsora <tushar.harsora.harsora@oracle.com>
Co-authored-by: Daniel Dietzler <36593685+danieldietzler@users.noreply.github.com>
Co-authored-by: Alex <alex.tran1502@gmail.com>

Donovoi committed:
Merge branch 'immich-app:main' into main

Donovoi commented:
hopefully that's all done!

Donovoi committed:
Merge branch 'immich-app:main' into main

Donovoi committed:
Merge branch 'immich-app:main' into main

zackpollard commented on cli/package.json:
```diff
@@ -59,7 +59,9 @@
   "dependencies": {
     "fast-glob": "^3.3.2",
-    "lodash-es": "^4.17.21"
+    "install": "^0.13.0",
```
You still have an extra dependency here that has been added erroneously it seems

Donovoi replied:
Awesome thank you for pointing it out. I removed it and re-ran npm install. hopefully its all fixed now 😁

Donovoi committed:
Merge branch 'immich-app:main' into main

Donovoi committed:
remove "install" package added accidentally

bo0tzz commented:
I think this has been superseded by #10430?
